improve-service-processing/README.md

```diff
@@ -0,0 +1,73 @@
+  [2022-04-12 21:44:06,445: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-34-x86_64': SRPM build is in progress...
+  [2022-04-12 21:44:06,794: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-rawhide-ppc64le': SRPM build is in progress...
+  [2022-04-12 21:44:07,169: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-35-s390x': SRPM build is in progress...
+  [2022-04-12 21:44:07,538: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-rawhide-x86_64': SRPM build is in progress...
+  [2022-04-12 21:44:07,870: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-35-aarch64': SRPM build is in progress...
+  [2022-04-12 21:44:08,215: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-36-x86_64': SRPM build is in progress...
+  [2022-04-12 21:44:08,541: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:centos-stream-9-aarch64': SRPM build is in progress...
+  [2022-04-12 21:44:08,893: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-34-s390x': SRPM build is in progress...
+  [2022-04-12 21:44:09,227: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:epel-8-x86_64': SRPM build is in progress...
+  [2022-04-12 21:44:09,576: DEBUG/ForkPoolWorker-1] Setting Github status check 'in_progress' for check 'rpm-build:fedora-rawhide-aarch64': SRPM build is in progress...
+  ```
+
+  - we probably can't do much about it
+
+- processing event when there is no config file
+  - finding out there is no config file takes few seconds (on average around 2, but this depends probably on the repo size,
+    I have seen also a log with 4 seconds duration)
+- the target expansion is also done more times during one processing, we could improve it (using properties and setting build/test targets only once),
+  but the process takes milliseconds, so this is not problematic from a time perspective
+
+## Suggestions for improvements
+
+#### Can we uninstall a repo? - Not receive events, after some period of time when we didn’t produce any meaningful work.
+
+- the documentation provides info on how to list installations, get a specific installation, delete, suspend it
+  (when a GitHub App is suspended, the app's access to the GitHub API or webhook events is blocked for that account)
+  https://docs.github.com/en/rest/reference/apps
+- we could have a periodic job which would check the installations and if an installed repo does not contain a config file
+  after some time limit (e.g. a month after installation), uninstall it
+  - if the app is installed on a namespace - at least one repo with a config file?
```

Comment:
:+1:

Reply:
agreed

since the workflow to enable packit again is quite clear: install it :)